react/components/ProductCustomizer.js

```diff
@@ -177,7 +187,8 @@ class ProductCustomizer extends Component {
    */
   handleVariationChange = async variationObject => {
     const { productQuery: { product } } = this.props
-    const variationSku = variation && variationObject.skuId
+    const variationSku = variationObject && variationObject.skuId
+
     const sku = product.items.find(sku => sku.itemId === variationSku)
 
     // TODO: add proper error message to handle null variationObject and sku
```

Comment:
This would be better as an issue.

Reply:
Created the issue: https://github.com/vtex-apps/product-customizer/issues/7

Will leave the todo here as well though, to find the code more easily later.